src/common/components/UserDetail/index.jsx

```diff
@@ -84,7 +84,7 @@ class UserDetail extends Component {
                 <div>{phoneLink || '--'}</div>
                 <div><span>&nbsp;</span></div>
                 <div>{user.chapter ? user.chapter.name : '--'}</div>
-                <div>{user.phase ? user.phase.number : '--'}</div>
+                <div>{user.phase ? user.phase.name : '--'}</div>
```

Comment:
`user.phase` is not in PropTypes. Should it be?

Reply:
Updated